src/libraries/System.Private.CoreLib/src/System/Globalization/Normalization.Windows.cs

```diff
@@ -51,10 +56,10 @@ internal static bool IsNormalized(string strInput, NormalizationForm normalizati
                     throw new InvalidOperationException(SR.Format(SR.UnknownError_Num, lastError));
             }
 
-            return result;
+            return result == Interop.BOOL.TRUE;
```

Comment:
Is a true result guaranteed to be TRUE?  Generally 0 is false and anything else can be true, so I'd expect this would be better as `result != Interop.BOOL.FALSE

Reply:
I don't know if it's guaranteed, docs only say TRUE and FALSE. Changed